plugins/pokescape-pvm

```diff
@@ -0,0 +1,4 @@
+repository=https://github.com/Quoded/PokeScape-PvM.git
+commit=baeb6822e5aa73b8adb6d7c55607d88188c955df
+warning=This plugin submits the names of you, loot, screenshots and other character data to api.pokescape.com
+authors=Quo
```

Comment:
```suggestion
warning=This plugin submits your IP address, RSN, world, loot, and screenshots to a 3rd party website not controlled or verified by the RuneLite Developers.
```

The `authors` line is not needed in this file, as it is only used here to help validate whether other users are authorized to submit plugin updates on your behalf.

Reply:
Done